project/dimensions/subsectors.csv

```diff
@@ -21,4 +21,28 @@ naics_311_312,Food and Beverage
 naics_325_other,Chemicals
 naics_3241,Petroleum Refineries
 naics_322,Pulp and Paper
-naics_3_other,Other Manufacturing
\ No newline at end of file
+naics_3_other,Other Manufacturing
+15_S,15 degrees tilt south azimuth
+25_S,25 degrees tilt south azimuth
+35_E,35 degrees tilt east azimuth
+25_SE,25 degrees tilt southeast azimuth
+25_W,25 degrees tilt west azimuth
+35_S,35 degrees tilt south azimuth
+15_SE,15 degrees tilt southeast azimuth
+25_SW,25 degrees tilt southwest azimuth
+45_E,45 degrees tilt east azimuth
+45_SE,45 degrees tilt southeast azimuth
+35_W,35 degrees tilt west azimuth
+35_SE,35 degrees tilt southeast azimuth
+15_E,15 degrees tilt east azimuth
+45_W,45 degrees tilt west azimuth
+25_E,25 degrees tilt east azimuth
+45_S,45 degrees tilt south azimuth
+15_SW,15 degrees tilt southwest azimuth
+55_S,55 degrees tilt south azimuth
+35_SW,35 degrees tilt southwest azimuth
+55_SE,55 degrees tilt southeast azimuth
+45_SW,45 degrees tilt southwest azimuth
+55_SW,55 degrees tilt southwest azimuth
+15_W,15 degrees tilt west azimuth
+55_E,55 degrees tilt east azimuth

```

Comment:
I don't think we need these at the project level? I think we should be aggregating these up to "Unspecified subsector" based on the weightings in the dGen load_data_lookup?